compose-builder/.env

```diff
@@ -39,14 +39,15 @@ DEVICE_LLRP_VERSION=latest
 DEVICE_COAP_VERSION=latest
 DEVICE_GPIO_VERSION=latest
 
-VAULT_VERSION=1.7.2
-CONSUL_VERSION=1.9.5
+# Note that Vault images don't have none patch versions like 1.8
+VAULT_VERSION=1.8.0
+CONSUL_VERSION=1.10
 # Note that Postgres images don't use patch versions.
-POSTGRES_VERSION=12.3-alpine
-REDIS_VERSION=6.2.4-alpine
-KONG_VERSION=2.4.1-alpine
+POSTGRES_VERSION=13.3-alpine
+REDIS_VERSION=6.2-alpine
+KONG_VERSION=2.5
```

Comment:
Verified that Kong never pushed the 2.5-alpine label and that it is just 2.5.

Reply:
Reverted back to 2.4